ledger-test-tool/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/CompanionImplicits.scala

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.suites.v1_8
+
+import com.daml.ledger.javaapi.data.codegen.ContractCompanion
+import com.daml.ledger.test.java.model.da.types
+import com.daml.ledger.test.java.model.iou.Iou
+import com.daml.ledger.test.java.model.test
+import com.daml.ledger.test.java.model.test.{
+  Agreement,
+  AgreementFactory,
+  CallablePayout,
+  Delegated,
+  Delegation,
+  Divulgence1,
+  Divulgence2,
+  Dummy,
+  DummyFactory,
+  DummyWithParam,
+  LocalKeyVisibilityOperations,
+  MaintainerNotSignatory,
+  ShowDelegated,
+  TextKey,
+  TextKeyOperations,
+  WithObservers,
+  Witnesses => TestWitnesses,
+}
+import com.daml.ledger.test.java.semantic.semantictests
+
+object CompanionImplicits {
+
+  implicit val dummyCompanion
+      : ContractCompanion.WithoutKey[Dummy.Contract, Dummy.ContractId, Dummy] = Dummy.COMPANION
```

Comment:
I have a feeling I have already looked at analogous companion. Should we think of deduplicating?

Reply:
f5d761992d567e23a6a1023901b5418989590674